test/click.spec.js

```diff
@@ -404,34 +404,48 @@ module.exports.describe = function({testRunner, expect, playwright, FFOX, CHROMI
       expect(await page.evaluate('window.clicked')).toBe(true);
     });
     xit('should click on an animated button', async({page}) => {
```

Comment:
nit: button animated by javascript.
CSS animations would be a very different thing.

Reply:
Turns out this is CSS animations AND setInterval